src/lib/GOBLBuilder.svelte

```diff
@@ -1,15 +1,36 @@
 <script lang="ts">
-  import MenuBar from "./ui/MenuBar.svelte";
-  import Toolbar from "./toolbar/Toolbar.svelte";
+  import { editor, envelope } from "$lib/stores.js";
+  import MenuBar from "./menubar/MenuBar.svelte";
   import Editor from "./editor/Editor.svelte";
 
-  export let jsonSchemaURL: string;
+  export let jsonSchemaURL = "";
+  export let value = "";
+
+  // When `value` is updated, update the internal editor and envelope stores.
+  // If `value` is JSON and it's a GOBL envelope, parse and store its contents
+  // and the envelope separately.
+  $: {
+    try {
+      const parsedValue = JSON.parse(value);
+      if (parsedValue.$schema === "https://gobl.org/draft-0/envelope") {
```

Comment:
Observation for the future... we'll change the versions eventually, you might want to stick this in a `isEnvelope` method that can use a list of known envelope versions.